backend/dal/Services/Admin/Concrete/ParcelService.cs

```diff
@@ -51,13 +51,20 @@ public Paged<Parcel> Get(ParcelFilter filter)
             var query = this.Context.Parcels.AsNoTracking();
```

Comment:
I like that you are adding better variable names but keep in mind that it makes it much harder to review what actually changed.